node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -104,6 +105,7 @@ class NodeRuntime(conf: Conf) {
 
   def nodeProgram(implicit scheduler: Scheduler): Effect[Unit] =
     for {
+      _         <- Log[Effect].info(s"  ============= RChain Node ${BuildInfo.version} ============".blue)
```

Comment:
All good, I would maybe remove the `=== ` and definitely remove `.blue` as this is going to logs and does not need any coloring at all (if anything this might be an issue to some log readers)

Reply:
If this is going to logs instead of stdout then I think it's not what @AbnerZheng wanted to do.

Reply:
@goral09 Yes, I mixed up these things.

Reply:
@rabbitonweb Done